keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/Environment.kt

```diff
@@ -1,6 +1,7 @@
 package com.netflix.spinnaker.keel.api
 
 data class Environment(
+  val application: String,
```

Comment:
I added this here because we were doing `environment.resources.first()?.application`, which was hacky and could return `null`.